### DISCUSSION TITLE
I have applied for sponsorship but yet I have not  getting  getting

### DISCUSSION THREAD

**Discussion by Alokrar** [community/community#30376] [Other Feature Feedback, Questions, & Ideas]
How

**Comment by melvincwng** [comment]
@Alokrar, for applications to GitHub Sponsors, typically you have to wait for a few weeks for your application to be approved.

Based on official [GitHub Documentation](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account), after application, GitHub would typically review your application within 2 weeks.

My suggestion would be to kindly wait a while and see if you receive an email (in the next few days to weeks) regarding your GitHub Sponsors application.

Good luck, and hope this answers your question 😊

**Reply by Alokrar** [reply]
Thanks for your suggestion I will wait for that..

**Comment by Bullin112** [comment]
Hi,
For what purpose did you apply for sponsorship? Was that for a [specific product](https://paintballnest.com/) or for account? Because for both purposes, procedures are different to go for and time limit is also changed for both. I think should contact through email at customer support or help center for the details. Thanks

**Reply by Alokrar** [reply]
It is for my account improvement.that also include some specific projects.
